.github/workflows/claude-nl-suite.yml

```
@@ -1,134 +1,521 @@
-      - name: Install Python + uv
-        uses: astral-sh/setup-uv@v4
-        with:
-          python-version: '3.11'
-
-      - name: Install UnityMcpServer deps
-        run: |
-          set -eux
-          if [ -f "UnityMcpBridge/UnityMcpServer~/src/pyproject.toml" ]; then
-
+    nl-suite:
+      if: github.event_name == 'workflow_dispatch'
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)